front/lib/api/assistant/actions/database_query.ts

```diff
@@ -282,8 +286,36 @@ export async function* runDatabaseQuery({
         return;
       }
 
+      if (event.content.block_name === "SQL") {
+        let tmpOutput = null;
+        if (e.value) {
+          const sql = e.value as string;
+          tmpOutput = { query: sql };
+        } else {
+          tmpOutput = { no_query: true };
+        }
+        yield {
```

Comment:
I could also yield this event after the block EXPLAIN and DATASET_DESCRIPTION if we want to build the output all along (displayed on the expanded Query chip) but this seemed overkill as: 
-> When no query is generated, we get the success event instantly after.
-> When a query is generated, we do not allow to explant the Query chip.

[Dust app](https://dust.tt/w/78bda07b39/a/e0c5993d65) for context.